freedomofpress/pressfreedomtracker.us
# Update database page incident card design

closes #1659 

Updates the incidents card on the database page to match the [figma designs](https://www.figma.com/file/H3uTXxT88ofhdebUFgzEo9/PFT-%E2%80%93-Hand-off-Guide?type=design&node-id=3555-37026&t=VpJgTRKijO6Hca3j-0)

Desktop screenshot:
![Screenshot 2023-05-30 at 8 37 03 PM](https://github.com/freedomofpress/pressfreedomtracker.us/assets/3477162/47237729-bcf9-49b3-8699-17b131d8bcdf)

Mobile screenshot:
![Screenshot 2023-05-30 at 8 37 33 PM](https://github.com/freedomofpress/pressfreedomtracker.us/assets/3477162/8f36909d-386d-4d29-9fa0-4e23f7302eb3)

## To test
Open the incidents database on both desktop and mobile viewports, and ensure that it matches the mocks. Also double check other areas of the application to ensure there are no accidental regressions.
